src/Version10/Definition/ExecTaskDefinitionBuilderInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpcq\PluginApi\Version10\Definition;
+
+use Phpcq\PluginApi\Version10\Definition\Builder\ConsoleApplicationBuilderInterface;
+
+interface ExecTaskDefinitionBuilderInterface
+{
+    /**
+     * Describes a console application provided by the plugin.
+     *
+     * If the plugin provides multiple console applications, a dedicated application name is required so that the
+     * application will be available as plugin:application. If no application name is provided the application is
+     * available using the plugin name.
+     */
+    public function describeApplication(
```

Comment:
I think we agreed that in the interfaces all parameters must be documented with their purpose - both are missing here.